AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -796,8 +796,24 @@ NS_ASSUME_NONNULL_BEGIN
 @end
 
 /*
- ASDisplayNode participates in ASAsyncTransactions, so you can determine when your subnodes are done rendering.
- See: -(void)asyncdisplaykit_asyncTransactionContainerStateDidChange in ASDisplayNodeSubclass.h
+ * ASDisplayNode support for automatic subnode management.
+ */
+@interface ASDisplayNode (AutomaticHierarchyManagement)
```

Comment:
Haha, no not at all! Just made fun of myself as I overlooked that. But should be updated now!